Tools/TestWebKitAPI/Tests/WebKitCocoa/SOAuthorizationTests.mm

```diff
@@ -621,11 +621,7 @@ static void checkAuthorizationOptions(bool userActionInitiated, String initiator
     Util::run(&authorizationPerformed);
     checkAuthorizationOptions(false, emptyString(), 0);
     EXPECT_FALSE(policyForAppSSOPerformed); // The delegate isn't registered, so this won't be set.
-#if PLATFORM(MAC)
     EXPECT_TRUE(gAuthorization.enableEmbeddedAuthorizationViewController);
-#elif PLATFORM(IOS) || PLATFORM(VISION)
```

Comment:
This seems wrong -- we set `enableEmbeddedAuthorizationViewController` to `NO` in `PLATFORM(VISION)` in `SOAuthorizationSession.mm`, so you should retain the assertion here.